launchable/commands/record/case_event.py

```diff
@@ -151,6 +154,17 @@ def stderr(case: TestCase) -> str:
     def create(cls, test_path: TestPath, duration_secs: float, status,
                stdout: Optional[str] = None, stderr: Optional[str] = None,
                timestamp: Optional[str] = None, data: Optional[Dict] = None) -> Dict:
+        def _timestamp(ts: Optional[str] = None):
+            if ts is None:
+                return datetime.datetime.now(datetime.timezone.utc).isoformat()
+            try:
+                date = dateutil.parser.parse(ts)
+                if date.tzinfo is None:
+                    return date.replace(tzinfo=UTC).isoformat()
```

Comment:
I wonder about fixing TZ to UTC... 
Depending on the setting of an environment where the CLI is executed, setting it to UTC by default might not be a good idea.

WDYT?

Reply:
Let me use a local time instead.

Reply:
Thanks